client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -913,7 +913,9 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 			<StepContainer
 				stepName={ STEP_NAME }
 				stepContent={ stepContent }
-				hideSkip
+				hideSkip={ ! isGoalsHoldout }
```

Comment:
I think we should use `isGoalCentricFeature` or it should be `isGoalsHoldout`, right?

Reply:
> from [this comment](https://github.com/Automattic/dotcom-forge/issues/10094#issuecomment-2548590658), This should only be for the hold out test! Thanks for confirming.

I negated to `hide the skip` when `isGoalsHoldout` is Disabled(false) - previous behaviour, and `show skip` when it is Enabled. @fditrapani could you please confirm.

Reply:
> https://github.com/Automattic/dotcom-forge/issues/10094#issuecomment-2547383295

Referring to [previous one](https://github.com/Automattic/dotcom-forge/issues/10094#issuecomment-2547383295), I assume it means we have to keep it hidden for people under the holdout test. I was confused about the similar description before 🙈

Reply:
Thanks you for clarifying 🙇 Updated the condition to hide skip when `isGoalsHoldout` is `true`

Reply:
Thanks for checking in. We only want to show this link within the new onboarding flow.